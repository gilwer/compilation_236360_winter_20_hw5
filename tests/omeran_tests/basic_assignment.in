// basic assignments with variables and literals
void main() {
     int x = 55;
     printi(x); // 55
     int y;
     printi(y); // 0
     y = x + 66;
     printi(y); // 121
     y = x + y;
     printi(y); // 176
     int z = x + 99;
     printi(z); // 154
     int w = y + x;
     printi(w); // 231
 }