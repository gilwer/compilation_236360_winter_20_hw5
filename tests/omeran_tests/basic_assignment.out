55
0
121
176
154
231
